docker-compose.yml

```diff
@@ -3,13 +3,27 @@
 version: '3.7'
 
 services:
-    app:
-        build:
-            context: .
-        ports:
-            - "8000:8000"
-        volumes:
-            - ./src:/delfitlm
-        command: sh -c "python manage.py runserver 0.0.0.0:8000"
-        environment:
-            - DEBUG=1
+  db:
+    image: postgres
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=${POSTGRES_DB:-postgres}
+      - POSTGRES_USER=${POSTGRES_DB:-postgres}
+      - POSTGRES_PASSWORD=${POSTGRES_DB:-postgres}
+  
+  app:
+    build:
+        context: .
+    ports:
+      - "8000:8000"
+    command: sh -c "python manage.py runserver 0.0.0.0:8000"
+    environment:
+        - DEBUG=1
+        - POSTGRES_DB=${POSTGRES_DB:-postgres}
+        - POSTGRES_USER=${POSTGRES_DB:-postgres}
+        - POSTGRES_PASSWORD=${POSTGRES_DB:-postgres}
+        - POSTGRES_HOST=${POSTGRES_DB:-db}
+        - POSTGRES_PORT=${POSTGRES_DB:-5432}
+    depends_on:
+      - "db"
```

Comment:
Just to double check docker takes the inputs for postgres from the `.env` file and if not provided they default to `postgres`, `db` and `5432`, right?

Reply:
The trick sounds nice but I see now we have 3 docker-compose files and the -deploy one has not been updated with the db container. The trick seems handy but I would also add a note in the README.md file about it to avoid forgetting it.

Reply:
I like the approach! I also like the fact you used pgadmin in the local development environment and I think it might be very useful for production as well. There was a discussion at a certain point about an eventual Django module acting as pgadmin (it could be cool) but otherwise, a "low-level" database manager will come in handy in the future. We might think about using a reverse proxy rule for Nginx but we might just expose the pgadmin port directly (bypassing Nginx) but not open the server firewall for that specific port (basically leaving pgadmin only accessible locally for security reasons). This might be food for future thoughts but, for now, we can go with the PR as-is for me.

Reply:
[DBeaver](https://dbeaver.io/) works nicely for remote database administration. I worked with it on another project and I was quite impressed with it.